scripts/run_python_test_script.py

```diff
@@ -65,10 +67,23 @@
 os.environ["MeshLibPyModulesPath"] = os.getcwd()
 os.chdir(directory)
 
+# remove meshlib package if installed to not shadow dynamically attached
 os.system(python_cmd + "-m pip uninstall -y meshlib")
+
+#command line to start test
 pytest_cmd = "-m pytest -s -v --basetemp=../pytest_temp --durations 30"
+if args.bindings_vers == '2':
+    pytest_cmd += ' -m "not bindingsV3'
+elif args.bindings_vers == '3':
+    pytest_cmd += ' -m "not bindingsV2'
```

Comment:
why use `-m not "bindingsV3`, mb use `-m bindingsV2` instead?

Reply:
Most of tests are untagged and works fine on both bindings.
"-m bindingsV2" will start only a few tests, that marked with this tag.